docs/automatic-installation.md

```diff
@@ -0,0 +1,255 @@
+# Automatic Installation
```

Comment:
Clarify that this is available as of v0.2.0

Reply:
Fixed. Thanks. I also include a link to https://github.com/harvester/ipxe-examples